.gitignore

```diff
@@ -1,2 +1,4 @@
 /lex.yy.c
 /cvsclone
+
+*.dSYM
```

Comment:
Please add a short comment above it with this information, if it's not a problem ("# OS X build artifact"). Thanks.